### ISSUE TITLE
Error in response_schema_conformance after update from 1.2.0 to 1.3.2

### ISSUE THREAD

**Issue by stepanbokoc** [kiwicom/schemathesis#539]
Hi,
In my schema I have `paths` and `schemas` in their own files under folder ./paths and ./schemas.

After update to 1.3.2 I started getting this exception 
`jsonschema.exceptions.RefResolutionError: <urlopen error [Errno 2] No such file or directory: '/home/sb/work/python/sraps/docs/application/openapi.yml'>
`

actual path is 
`/home/sb/work/python/sraps/docs/application/openapi/openapi.yml`

example of `paths` file
```
description: Desc
get:
  responses:
    200:
      description: description
      content:
        application/json:
          schema:
            type: array
            items:
              $ref: "../openapi.yml#/components/schemas/User"
```

The problem here is with `../`. It looks for `"../openapi.yml#/components/schemas/User"` from base directory (`/home/sb/work/python/sraps/docs/application/openapi/`) and not from ./paths (`/home/sb/work/python/sraps/docs/application/openapi/paths/`)

**Comment by Stranger6667** [comment]
Hi!

Thank you for reporting this. The bug is connected with the recent changes in `response_schema_conformance` check - now it uses a not-resolved schema. I will investigate

**Comment by Stranger6667** [comment]
Ok, I added one more bug there - `_get_response_schema` now accepts not resolved schema, and it is not resolved inside. After adding resolving, it fails with an error :/ I will work on a fix

**Comment by Stranger6667** [comment]
@stepanbokoc 

The fix is released in [1.4.0](https://pypi.org/project/schemathesis/1.4.0/)

Let me know if it works for your case

**Comment by stepanbokoc** [comment]
Yes it works now, thank you sir
